CONTRIBUTING.md

```diff
@@ -219,3 +222,14 @@ The footer should contain any information about **Breaking Changes** and is also
 [coc]: https://github.com/modflowpy/flopy/blob/develop/CODE_OF_CONDUCT.md
 [github]: https://github.com/modflowpy/flopy
 [stackoverflow]: https://stackoverflow.com/questions/tagged/flopy
+
+## <a name="coc"></a> Pre-commit hooks
```

Comment:
Maybe worth linking here from `DEVELOPER.md` (or moving there) as it seems like something contributors may want to do when first setting up a development environment?

Reply:
@w-bonelli I moved the install instructions to `DEVELOPER.md` and then linked back to them.